os/mm/kmm_heap/kmm_malloc.c

```diff
@@ -137,8 +137,9 @@ void *kmm_malloc_at(int heap_index, size_t size)
 
 	kheap = kmm_get_heap();
 #ifdef CONFIG_DEBUG_MM_HEAPINFO
-	ARCH_GET_RET_ADDRESS
-	ret = mm_malloc(&kheap[heap_index], size, retaddr);
+	size_t caller_retaddr;
+	ARCH_GET_RET_ADDRESS(caller_retaddr)
+	ret = mm_malloc(&kheap[heap_index], size, caller_retaddr);
 	if (ret == NULL) {
```

Comment:
Line 143 ~ 146 and 149 ~ 152  are duplicated. Let's get them out of the config.

Reply:
Modified. Thanks.